frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx

```diff
@@ -76,14 +76,17 @@ export const EncryptionFormGroup: React.FC<EncryptionFormGroupProps> = ({
   }, [encryption.clusterWide, encryption.storageClass, encryptionChecked]);
```

Comment:
I can see a separate useEffect is to set hasHandled for all cases, Setting a flag is kind of redundant and leads to unpredicted behavior, so I removed it from my latest change